### DISCUSSION TITLE
نسخه اندروید (1.0.8) هیچ نتیجه ای بر نمیگردونه

### DISCUSSION THREAD

**Discussion by taregh-n** [MortezaBashsiz/CFScanner#579] [General]
سلام و تشکر از زحماتتون
نسخه **اندروید** نمیدونم چرا هیچ نتیجه ای رو بر نمیگردونه در حالی که نسخه ویندوز با همون رنج آی پی و با همون اینترنت نتایج خوبی بر میگردونه! 
قبلا نسخه ویندوز مشکل فرانتیگ رو داشت و چیزی بر نمیگردوند ولی توی نسخه جدیدش عالی شده، منتظر آپدیت اندروید بودم که شاید توی نسخه جدید مث ویندوز مشکلش حل بشه ولی فرقی نکرده و بازم چیزی بر نمیگردونه
ممنون میشم اگه نکته خاصی داره راهنمایی کنید

**Comment by boyluti** [comment]
الان نسخه ویندوز اصلا کار نمیکنه. تا 2 روز پیش کار میکرد. الان همش ارور
